zaza/openstack/utilities/generic.py

```diff
@@ -139,21 +139,35 @@ def get_undercloud_env_vars():
     :returns: Network environment variables
     :rtype: dict
     """
-    # Handle backward compatibile OSCI enviornment variables
+    # Handle OSCI environment variables
+    # Over time we have changed the names of prefixes. Try them all in
+    # ascending order of significance.
+    # Note: TEST_* is the preferred prefix
+    _prefixes = ["AMULET_", "MOJO_", "OS_", "", "TEST_"]
```

Comment:
I'd rather not introduce a NONE prefix.  I'd rather make this reflect the expected realty of passenv, which is:

```
passenv = HOME TERM CS_* OS_* TEST_*
```